web/web.go

```diff
@@ -240,9 +241,16 @@ func MustBuildServerURL(pathPrefix string) string {
 	if err != nil {
 		panic(err)
 	}
-	hostname, err := os.Hostname()
+	hostname, err := getServerHostname()
 	if err != nil {
 		panic(err)
 	}
 	return fmt.Sprintf("http://%s:%s%s", hostname, port, pathPrefix)
 }
+
+func getServerHostname() (string, error) {
+	if *hostname != "" {
+		return *hostname, nil
+	}
+	return os.Hostname()
+}
```

Comment:
I'm using vim with a ton of Go plugins and other plugins (Ctrl-P is awesome for fuzzy-finding for example: https://github.com/kien/ctrlp.vim, NERDTree for a file tree, etc.).